public/js/admin.js

```diff
@@ -436,7 +437,10 @@ var updateVisibility = function(qid) {
         },
         success: function(data) {
             displayQuestionTable();
-            dropSnack(colours.SUCCESS_GREEN, 'Question ' + qid + ' is now visible to the students');
+            var msg = question['visible'] ? ' is now visible to the students' : ' is now&nbsp<u><b>not</b></u>&nbspvisible to the students';
```

Comment:
any reason why we are using var over const?

Reply:
done.